packages/ui/src/components/vuestic-components/va-dropdown/VaDropdown.vue

```diff
@@ -128,15 +117,34 @@ export default class VaDropdown extends mixins(DropdownPropsMixin) {
     })
   }
 
+  handleClick (emitName: ClickType, toClose: boolean): void {
+    this.$emit(emitName)
+    if (toClose) {
```

Comment:
`toClose` doesn't seem to be useful, you can just use `this.closeOnClickInside` here directly.